### PR TITLE
[10.0][ADD] web_tab_focus

### PR DESCRIPTION
**Pull request by andreparames**
This is a proposal for a module that allows a page/tab of a notebook to be automatically focused depending on the value of a given field of the record. I suggest reading the [README](https://github.com/acsone/web/blob/85e08a7e431544ace5551eb2e80ac2123ea8e2f7/web_tab_focus/README.rst). Suggestions regarding the syntax are welcome!

### CODE REVIEW

File: web_tab_focus/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    'version': '10.0.1.0.0',
+    'license': 'AGPL-3',
+    'depends': [
+        'web',
+    ],
+    'data': [
+        'views/templates.xml',
+    ],
+    "demo": [
+    ],
```
Comment (tarteo) [inline web_tab_focus/__manifest__.py]: Can you remove the demo key? it's empty

**Review by tarteo — APPROVED** [review]
Tested it on Runbot. 
It works great and is great to improve user experience. Thanks!

File: web_tab_focus/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# coding: utf-8
+# Copyright 2018 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': "Web Tab Focus",
+    'author': 'ACSONE SA/NV,Odoo Community Association (OCA)',
+    'website': "https://odoo-community.org",
```
Comment (simahawk) [inline web_tab_focus/__manifest__.py]: https://github.com/OCA/web

File: web_tab_focus/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    'license': 'AGPL-3',
+    'depends': [
+        'web',
+    ],
+    'data': [
+        'views/templates.xml',
+    ],
+    "demo": [
+    ],
+    'application': False,
```
Comment (simahawk) [inline web_tab_focus/__manifest__.py]: this too

**Review by simahawk — CHANGES_REQUESTED** [review]

**Comment by andreparames** [comment]
Thanks for the suggestions @tarteo and @simahawk. I've pushed an update.

**Review by simahawk — APPROVED** [review]
LGTM. Travis is red because of bokeh broken dependency.
@lreficent have you find a solution? Seems the same bug you have here https://github.com/OCA/web/pull/889

**Comment by andreparames** [comment]
@simahawk @lreficent I got Travis to use Python 2.7.13 by using https://github.com/praekeltfoundation/travis-pyenv, which fixes the bokeh dependency.

I'm not sure if this should be merged to the main repo, though.

**Comment by LoisRForgeFlow** [comment]
I actually opened a issue some time ago: https://github.com/OCA/web/issues/893

Please report your opinion there, and see if we get some expert feedback.

**Comment by simahawk** [comment]
@andreparames can you rebase? the issue should be fixed now

**Comment by andreparames** [comment]
@simahawk thanks for the heads-up, I've rebased.

**Review by MeetKD — APPROVED** [review]
LGTM

**Pull request closed without merging**